src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -124,19 +115,16 @@ While it is possible to install those charts separately, we strongly recommend u
 
   Please notice and adjust the following flags:
 
-
-  * `global.licenseKey=YOUR_NEW_RELIC_LICENSE_KEY`: Must be set to a valid License Key for your account.
-  * `global.cluster=K8S_CLUSTER_NAME`: Is used to identify the cluster in the New Relic UI, so should be a descriptive value not used by any other Kubernetes cluster configured in your New Relic account.
-  * `kube-state-metrics.enabled=true`: Setting this to `true` will automatically install Kube State Metrics (KSM) for you, which is required for our integration to run. You can set this to false if KSM is already present in your cluster, even if it is on a different namespace.
-  * `newrelic-prometheus-agent.enabled=true`: Will deploy our Prometheus Agent, which automatically collects data from Prometheus endpoints present in the cluster.
-  * `nri-metadata-injection.enabled=true`: Will install our minimal webhook, which adds environment variables that, in turn, allows [linking applications instrumented with New Relic APM to Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes).
-
+    * `global.licenseKey=YOUR_NEW_RELIC_LICENSE_KEY`: Must be set to a valid License Key for your account.
+    * `global.cluster=K8S_CLUSTER_NAME`: Is used to identify the cluster in the New Relic UI, so should be a descriptive value not used by any other Kubernetes cluster configured in your New Relic account.
+    * `kube-state-metrics.enabled=true`: Setting this to `true` will automatically install Kube State Metrics (KSM) for you, which is required for our integration to run. You can set this to false if KSM is already present in your cluster, even if it is on a different namespace.
+    * `newrelic-prometheus-agent.enabled=true`: Will deploy our Prometheus Agent, which automatically collects data from Prometheus endpoints present in the cluster.
+    * `nri-metadata-injection.enabled=true`: Will install our minimal webhook, which adds environment variables that, in turn, allows [linking applications instrumented with New Relic APM to Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes).
 
   Our Kubernetes charts have a comprehensive set of flags and tunables that can be edited to better fit your particular needs. Please check the [Configure the integration](#configure) section below to see what can be changed.
 
 5. Install the Kubernetes integration by running the command without `--debug` and `--dry-run`:
 
-
   ```shell
   helm upgrade --install newrelic newrelic/nri-bundle \
```

Comment:
We change this so that it's consistent with the guided install helm release name of `newrelic-bundle`.
```
helm upgrade --install newrelic-bundle newrelic/nri-bundle \
```